<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QListWidget" name="ListOfTutorials">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>731</width>
     <height>233</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>279</y>
     <width>731</width>
     <height>232</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="TitleLabel">
      <property name="text">
       <string>Title</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="TitlelineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="PresenterLabel">
      <property name="text">
       <string>Presenter</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="PresenterlineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="DurationLabel">
      <property name="text">
       <string>Duration</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="MinutesLabel">
      <property name="text">
       <string>Minutes</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="MinuteslineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="SecondsLabel">
      <property name="text">
       <string>Seconds</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="SecondslineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="LikesLabel">
      <property name="text">
       <string>Likes</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="LikeslineEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="LinkLabel">
      <property name="text">
       <string>Link</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="LinklineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>517</y>
     <width>731</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="AddButton">
      <property name="text">
       <string>Add a tutorial</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="DeleteButton">
      <property name="text">
       <string>Delete a tutorial</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="UpdateButton">
      <property name="text">
       <string>Update a tutorial</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ListButton">
      <property name="text">
       <string>List All Tutorials</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AddButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>AddTutorial()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>543</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>565</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DeleteButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>DeleteTutorial()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>537</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>565</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UpdateButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>UpdateTutorial()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>517</x>
     <y>535</y>
    </hint>
    <hint type="destinationlabel">
     <x>518</x>
     <y>570</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ListButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>ListAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>715</x>
     <y>544</y>
    </hint>
    <hint type="destinationlabel">
     <x>722</x>
     <y>576</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>AddTutorial()</slot>
  <slot>DeleteTutorial()</slot>
  <slot>UpdateTutorial()</slot>
  <slot>ListAll()</slot>
 </slots>
</ui>
